Commands/ImportReports.php

```diff
@@ -418,4 +427,18 @@ private function isValidDate($date)
             return false;
         }
     }
+
+    public function checkIfCanProcess()
+    {
+        $nextAvailableAt = Option::get(GoogleAnalyticsQueryService::DELAY_OPTION_NAME);
+        if ($nextAvailableAt === false) {
+            return ['canProcess' => true];
+        }
+
+        if(Date::factory('now')->getTimestamp() >= $nextAvailableAt){
+            Option::delete(GoogleAnalyticsQueryService::DELAY_OPTION_NAME);
+            return ['canProcess' => true];
+        }
+        return ['canProcess' => false, 'nextAvailableAt' => date('Y-m-d h:i a', $nextAvailableAt)];
```

Comment:
The `nextAvailableAt` is using `date` with `Y-m-d h:i a` and in `ImporReportsGA4.php` its using `strtotime` with `Y-m-d` can we keep it consistent at both the place ?

Reply:
Updated